src/main/kotlin/lotto/domain/WinningLotto.kt

```diff
@@ -2,17 +2,28 @@ package lotto.domain
 
 import lotto.domain.vo.LottoNumber
 
-class WinningLotto {
+class WinningLotto(winningNumbers: List<Int>, bonusNumber: Int) {
     private val winningLotto: Lotto
 
-    constructor(winningNumbers: List<Int>) {
+    private val bonusNumber: LottoNumber
+
+    private val winningLottoNumbers: Set<LottoNumber>
+        get() = winningLotto.lottoNumbers
+
+    init {
+        require(winningNumbers.contains(bonusNumber).not()) { "보너스번호는 로또 번호와 중복될수 없습니다." }
```

Comment:
```suggestion
        require(!winningNumbers.contains(bonusNumber)) { "보너스번호는 로또 번호와 중복될수 없습니다." }
```
위처럼 구현할수도 있지만 not() 함수를 사용해보았는데, 마찬가지로 일반적인 구현방법인지 판단이 어렵네요. 
not함수를 사용함에 따라 가독성이 좋아진것 같은데 현석님의 의견이 궁금합니다 😄

Reply:
이 부분 또한 취향차이일 것 같아요.
저같은 경우에는 느낌표가 잘 보이지 않는 경우에는, not을 활용해요.

예를 들면, lateinit var 를 사용하는 경우가 있는데요. (변수명을 a로 가정)
이 때 초기화 체크를 ::a.isInitialized 를 사용해서 초기화 여부를 판단하는데요.
not을 붙인다면
- !::a.inInitailzed
- ::a.isInitialized.not()
이 두가지 경우가 있는데, 저는 후자가 더 보기 좋아서 이 때에는 not을 활용합니다.